java/gandiva/pom.xml

```diff
@@ -111,6 +111,7 @@
             <resource>
                 <directory>${gandiva.cpp.build.dir}</directory>
                 <includes>
+                    <include>**/gandiva_jni.*</include>
```

Comment:
which are these files ? does windows generate files without the 'lib' prefix ?

Reply:
@pravindra 

Yes, 'gandiva_jni.dll' is generated on Windows.

```
D:\development\repository\git\arrow\cpp (ARROW-5132 -> origin)
$ ls -l build\release\
total 131600
-rwxr-xr-x 1 masayuki なし   295936 Apr  8 00:01 arrow-file-to-stream.exe*
-rw-r--r-- 1 masayuki なし      381 Apr  8 00:01 arrow-file-to-stream.exe.manifest
-rwxr-xr-x 1 masayuki なし   292864 Apr  8 00:01 arrow-stream-to-file.exe*
-rw-r--r-- 1 masayuki なし      381 Apr  8 00:01 arrow-stream-to-file.exe.manifest
-rwxr-xr-x 1 masayuki なし  2956288 Apr  8 00:01 arrow.dll*
-rw-r--r-- 1 masayuki なし      381 Apr  8 00:01 arrow.dll.manifest
-rw-r--r-- 1 masayuki なし   937846 Apr  8 00:01 arrow.exp
-rw-r--r-- 1 masayuki なし  1513600 Apr  8 00:01 arrow.lib
-rw-r--r-- 1 masayuki なし 58936526 Apr  7 23:58 arrow_static.lib
-rwxr-xr-x 1 masayuki なし 21671424 Apr  8 00:03 gandiva.dll*
-rw-r--r-- 1 masayuki なし      381 Apr  8 00:03 gandiva.dll.manifest
-rw-r--r-- 1 masayuki なし   186070 Apr  8 00:03 gandiva.exp
-rw-r--r-- 1 masayuki なし   302496 Apr  8 00:03 gandiva.lib
-rwxr-xr-x 1 masayuki なし 21915136 Apr  8 00:05 gandiva_jni.dll*
-rw-r--r-- 1 masayuki なし      381 Apr  8 00:05 gandiva_jni.dll.manifest
-rw-r--r-- 1 masayuki なし     3817 Apr  8 00:05 gandiva_jni.exp
-rw-r--r-- 1 masayuki なし     6952 Apr  8 00:05 gandiva_jni.lib
-rw-r--r-- 1 masayuki なし  3482426 Apr  8 00:04 gandiva_jni_static.lib
-rw-r--r-- 1 masayuki なし 22213694 Apr  8 00:04 gandiva_static.lib
```